### ISSUE TITLE
Adding Partial Slurs not Accessible for Screen Readers

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#27908]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Currently it is not possible to add a partial slur before a repeat using only keyboard.

STEPS:
1.  Create a few bars of music with a repeat sign.
2. Using arrow keys move focus to the note before the repeat.
3. Hold down shift and press right arrow.
4. The selection moves to the next note or rest after the repeat and there is no way to stop on the repeat to add the slur.
5. Compare standard way to input.  Click on the note before the repeat, Ctrl click the repeat sign and press S for slur.

Suggestion from Marc Sabatella that if repeat signs are added to the list of items along with notes and rests that can be selected with Shift and arrows then this slur could be added.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5.2

### Regression

No.

### Operating system

Windows 11

### Additional context

Using NVDA screen reader

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
